app/uk/gov/hmrc/customs/managesubscription/connectors/CustomsDataStoreConnector.scala

```diff
@@ -34,10 +33,17 @@ class CustomsDataStoreConnector @Inject() (appConfig: AppConfig, httpClient: Htt
   ec: ExecutionContext
 ) {
 
+  private val logger = Logger(this.getClass)
+
   def storeEmailAddress(dataStoreRequest: DataStoreRequest)(implicit hc: HeaderCarrier): Future[HttpResponse] = {
     val query =
       s"""{"query" : "mutation {byEori(eoriHistory:{eori:\\"${dataStoreRequest.eori}\\"}, notificationEmail:{address:\\"${dataStoreRequest.email}\\", timestamp:\\"${dataStoreRequest.emailVerificationTimestamp}\\"})}"}"""
     val header = hc.copy(authorization = Some(Authorization(s"Bearer ${appConfig.customDataStoreToken}")))
+    val url    = appConfig.customDataStoreUrl
+
+    // $COVERAGE-OFF$Loggers
+    logger.debug(s"[StoreEmailAddress: $url, body: $query and headers: $header")
```

Comment:
You missed closing `]` or you forgot to remove the opening one `[`

Reply:
But it's not that important because we need to remove this CustomsDataStore calls